packages/deployer/src/cli.ts

```diff
@@ -45,7 +45,7 @@ async function onDeployCommandAsync(argv: CliOptions): Promise<void> {
     const web3Wrapper = new Web3Wrapper(web3Provider);
     const networkId = await web3Wrapper.getNetworkIdAsync();
     const compilerOpts: CompilerOptions = {
-        contractsDir: argv.contractsDir,
+        contractDirs: getContractDirectoriesFromList(argv.contractsDir),
```

Comment:
We should add a regression test for this.

Reply:
Looks like there are not explicit tests for the command line interface.

Presumably, this is because compiling other dependent packages (Deployer, Contracts, Metacoin) would have failed if the CLI is wrong. Although, in this particular case, I had been working on all of those packages and would have built them manually at some point. So, we definitely do need explicit explicit test frameworks for all of our CLI's.